src/components/resetPassword/resetPasswordForm.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```

Comment:
@ValerieRono to increase your coverage, you can test this component using mount

Reply:
The coverage report says this component has been tested 100% by way of testing the view that renders the form, I have added tests to the utils file